README.md

```diff
@@ -0,0 +1,14 @@
+# DiscoUltimate & DiscoUtlimate v2 TMC - Klipper Config
+
+**!!! Work in Progress !!!**
+
+The purpose of this repository is to build a klipper config for the DiscoUltimate & DiscoUltimate v2 TMC.  
+The repository is supervised by Dagoma but the main contributions come from the community.  
+
+**Dagoma do not take any responsibility for this code. They aren't responsible of any issues or damages which may occur. Use at your own risk.**
```

Comment:
I don't think it necessary to specify this as it is explicitly mentioned the klipper configs provided are made by the Dagoma community.

```suggestion
~**Dagoma do not take any responsibility for this code. They aren't responsible of any issues or damages which may occur. Use at your own risk.**~
```